docs/src/main/sphinx/security/security-overview.rst

```diff
@@ -0,0 +1,159 @@
+=================
+Security overview
+=================
+
+After the initial :doc:`installation </installation>` of your cluster, security
+is the next major concern for successfully operating Trino. This overview
+provides an introduction to different aspects of configuring security for your
+Trino cluster.
+
+Aspects of configuring security
+-------------------------------
+
+The default installation of Trino has no security features enabled. Security
+can be enabled for different parts of the Trino architecture:
+
+* :ref:`security-client`
+* :ref:`security-inside-cluster`
+* :ref:`security-data-sources`
+
+Suggested configuration workflow
+--------------------------------
+
+To configure security for a new Trino cluster, follow this best practice
+order of steps. Do not skip or combine steps.
+
+1. **Enable** HTTPS
+
+* Work with your security team.
+* Use a load balancer or proxy to terminate HTTPS, if possible.
+* Use a globally trusted TLS certificate.
+
+2. **Enable authentication**
+
+* Start with a :doc:`password file <password-file>` to get up and running.
```

Comment:
password file authentication

Reply:
OK, but starting with a _file_ sounds much less scary to new users than starting with the entire concept of Password file authentication.